ResourceValidation.ipynb

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
This is the start of some work to explore validation and validation errors. Very basic data-wise for now.

Reply:
nice. send me an update when it's ready for me to look at visually!